scss/_patterns_breadcrumbs.scss

```diff
@@ -7,11 +7,13 @@
     margin: 0;
     padding: 0;
     width: 100%;
+  }
 
-    &__items {
-      margin-left: 0;
-      padding-left: 0;
-    }
+  .p-breadcrumbs__items {
+    @extend %small-caps-text;
```

Comment:
I think it would feel more natural for text styles to be moved on text element `p-breadcrumbs__item` instead. Especially that currently `p-breadcrumbs__item` in line 20 extends `%default-text` which conflicts in some styles with `%small-caps-text`, so it would be clearer if `__item` styles only extend `%small-caps-text` and nothing else.